src/shared/vscode/env.ts

```diff
@@ -88,3 +88,10 @@ export function isMinimumVersion(): boolean {
 export function getMinVscodeVersion(): string {
     return packageJson.engines.vscode.replace(/[^~]/, '')
 }
+
+/**
+ * Returns the minimum nodejs version declared in `package.json`.
+ */
+export function getMinNodejsVersion(): string {
+    return packageJson.devDependencies['@types/node'].replace(/[^~]/, '')
```

Comment:
not ideal, but this informally (now formally) indicates what nodejs version we are targeting

Reply:
You can add `node` to the `engines` section.

Reply:
based on https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines , `engines` is only advisory. So I'll skip it for now, since that would be an extra place to update.